app/models/host.rb

```diff
@@ -17,8 +20,13 @@ class Host < ApplicationRecord
   has_many :policies, through: :policy_hosts
   has_many :assigned_profiles, through: :policies, source: :profiles
 
-  validates :name, presence: true
-  validates :account, presence: true
+  def readonly?
+    true
+  end
```

Comment:
This is good for detecting issues with our tests, but also it should allow active_record to warn of deletion/update/insert events rather than postgres (can't modify records from a view).